src/classic/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, StoryFn } from "@storybook/react"
+import Button from "./Button"
+import { ButtonProps } from "~/common"
+
+export default {
+  title: "Classic/Components/Button",
+  component: Button,
+  argTypes: {},
+} as Meta
+
+// Template for stories
+const Template: StoryFn<ButtonProps> = (args) => <Button {...args} />
```

Comment:
nit: would be best to pick only the props that are used here so the docs don't show the other unused props